### ISSUE TITLE
Klona error Maximum call stack size exceeded

### ISSUE THREAD

**Issue by kevin4dhd** [open-wa/wa-automate-nodejs#2383]
### Are you using the latest version of the library?

- [X] I have checked and am using the latest version of the library.

### Are you using Multi-device?

- [X] I am using a multi-device session and I have set multiDevice to true in my config OR am using the --multi-device flag

### Current Behavior

After some messages, the bot stops working, an error occurs in the klona library, this causes the bot to stop and close

### Expected Behavior

The bot is expected to continue to function normally

### Steps To Reproduce

Start the bot in windows 10, send a couple of messages and the error will appear

### Mode

My own code

### create() code

```markdown
wa.create(
    {
        //useChrome: false,
        executablePath: 'C:\\Program Files (x86)\\Google\\Chrome\\Application\\chrome.exe',
        blockCrashLogs: true,
        disableSpins: true,
        logConsoleErrors: true,
        useStealth: true,
        corsFix: true,
        sessionId: process.env.PATH_SESION,
        qrTimeout: 0,
    }
).then(client => start(client));
```

### DEBUG INFO

```markdown
- Debug info: {
  "WA_VERSION": "2.2147.16",
  "PAGE_UA": "WhatsApp/2.2147.16 Mozilla/5.0 (Macintosh; Intel Mac OS X 11_2_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.90 Safari/537.36",
  "WA_AUTOMATE_VERSION": "4.28.0",
  "BROWSER_VERSION": "HeadlessChrome/96.0.4664.110",
  "OS": "Windows 10",
  "START_TS": 1640842764887
}
```

### Environment

```markdown
- OS: Windows 10
- Node: v16.13.0
- npm: 8.2.0
```

### Screenshots

![image](https://user-images.githubusercontent.com/22886367/147725094-4fb1706a-3acb-48ed-92b5-de131f983ed7.png)
![image](https://user-images.githubusercontent.com/22886367/147725096-7812074e-3f4b-4ca3-81e4-12530a99b1b9.png)

### Anything else?

I am not using "Multi-device" 
it won't let me create the issue without checking this option

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.28.1](https://github.com/open-wa/wa-automate-nodejs/compare/4.28.0...4.28.1) (2021-12-30)

- 🐛 fix #2382 [`#2382`](https://github.com/open-wa/wa-automate-nodejs/issues/2382)
- Release 4.28.1 [`6d3508a`](https://github.com/open-wa/wa-automate-nodejs/commit/6d3508a026ce2e83767c92dd97c0c6bac38820b3)
- 🚑 logs: prevent `Max call stack` klona error #2383 [`ffc18f5`](https://github.com/open-wa/wa-automate-nodejs/commit/ffc18f5f6bd4b366ee458fb6c5ee22e7cd16e400)
- updated types-only package [`9877ae2`](https://github.com/open-wa/wa-automate-nodejs/commit/9877ae23533def5f42b2933379db627c664db8b2)
